services/paybuttonsService.ts

```diff
@@ -10,9 +10,9 @@ export interface CreatePaybuttonInput {
   prefixedAddressList: string[]
 }
 
-export async function createPaybutton (userId: string, prefixedAddressList: string[]): Promise<Paybutton> {
+export async function createPaybutton (values: CreatePaybuttonInput): Promise<Paybutton> {
```

Comment:
~~The API wasn't updated to use this new format, but no test catches that.

- Add a test that will fail (does the paybutton actually get created?)
- Update the API to use this new `createPayButton` argument format

EDIT: There is another test that fails in the integration test. After I fixed that, the integration test now fails to create the paybutton, so ignore the previous test feedback. Just update the API~~

I see that all the API stuff is added in a separate commit. This looks good to go then